test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -66,6 +66,10 @@ class test_ucp_sockaddr : public ucp_test {
     ucs::sock_addr_storage m_test_addr;
 
     void init() {
+        if (!cm_use_all_devices() && has_transport("dc_x")) {
+            UCS_TEST_SKIP_R("DC+UD:aux doesn't support CM_USE_ALL_DEVICES=n");
```

Comment:
why it doesn't support?

Reply:
can you pls add comment that DC+UD requires ALL_DEVICES=y because it does not fi to prive data